src/trivyHelper.ts

```diff
@@ -57,8 +59,12 @@ export async function runTrivy(): Promise<TrivyResult> {
 }
 
 export async function getTrivy(): Promise<string> {
-    const latestTrivyVersion = await getLatestTrivyVersion();
 
+    let latestTrivyVersion = inputHelper.trivyVersion;
```

Comment:
Rename the var to just `trivyVersion`. `latestTrivyVersion ` is misleading here

Reply:
renamed to "version"